### ISSUE TITLE
Make readme.md less Hollywood, and more real CLI examples

### ISSUE THREAD

**Issue by englishextra** [anodynos/uRequire#73]
That's the first something with README that reads Much Ado, and nothing specific. Needs rewriing.

**Comment by anodynos** [comment]
Yeah, you have a point - the docs have moved to http://urequire.org/ anyway, but those need a heavy revising too... 

Unfortunately, since I have limited time and there are browserify, webpack, systemjs, and myriad other build/bundling projects out there, I am not committed to urequire anymore :-(
